.travis.yml

```diff
@@ -57,6 +57,19 @@ matrix:
     services:
       - postgresql
 
+  - python: 3.6
```

Comment:
we're excluding python 3.6 deliberately: we get four workers on Travis, so this one just slows things down pointlessly.

Reply:
ah, sorry. I think this one snuck during the rebase, didn't mean to add it